cloudnatix/README.md

```diff
@@ -0,0 +1,53 @@
+# Agent Check: CloudNatix
+
+## Overview
+
+This check provides the data from [CloudNatix][1].
```

Comment:
The overview section should include: 

1. What does Cloudnatix do in 1-2 sentences? 
2. What will customers get from this integration? 
3. What types of data is sent in and what is the value of the data sent in?

Reply:
Done